Source/WebCore/rendering/RenderViewTransitionCapture.cpp

```diff
@@ -46,6 +46,10 @@ RenderViewTransitionCapture::~RenderViewTransitionCapture() = default;
 void RenderViewTransitionCapture::setImage(RefPtr<ImageBuffer> oldImage)
 {
     m_oldImage = oldImage;
+    if (hasLayer())
+        layer()->contentChanged(ContentChangeType::Image);
+    if (parent())
+        repaint();
```

Comment:
Is this related?

Reply:
It is! Updated the commit message to explain it better.